.travis.yml

```diff
@@ -28,6 +28,11 @@ os:
   - linux
   - osx
 
+addons:
+  apt:
+    packages:
+      - libsecret-1-dev
```

Comment:
Do we care about which version of libsecret that we depend on? Also, is the package really called libsecret-1-dev, with the 1 in the middle?

Reply:
I believe that is what the package is called. I am reflecting what VSCode uses in their .travis.yml

https://github.com/Microsoft/vscode/blob/master/.travis.yml